website/docs/client/java/partials/java_circuit_breaker.md

```diff
@@ -121,18 +121,38 @@ it has a cachedthreadpool, which can result in creation of too many threads and
 It also rejects a task right away if there's no thread to pick up.
 
 <Tabs groupId="async">
-<TabItem value="flink" label="Flink Config">
+<TabItem value="yaml" label="Yaml Config">
+
+```yaml
```

Comment:
Just a side note: I wonder if could ever generate (not necessarily automatically) those config examples.

Reply:
What bothers me even more is each typo change shows whole table as diff in git, which makes it really difficult to review properly.